scripts/delete-old-resourcegroups.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+PREFIX=''
+KEEP_AGE=0
+
+print_usage() {
+  echo "Usage: delete-old-resourcegroups.sh -p <RESOURCE_PREFIX> [-a <KEEP_AGE>]"
+  echo "    -p (required): Delete all resource groups with this prefix"
+  echo "    -a: Only delete the resource group if it is older than this many seconds. Default is 0 which means delete all matching resource groups."
+}
+
+while getopts 'p:a:' flag; do
+  case "${flag}" in
+    p) PREFIX="${OPTARG}" ;;
+    a) KEEP_AGE=${OPTARG} ;;
+    *) print_usage
+       exit 1 ;;
+  esac
+done
+
+
+if [ ! $PREFIX ]; then
+  echo "-p flag is required"
+  print_usage
+  exit 1
+fi
+
+# Verify that the KEEP_AGE is an int
```

Comment:
It might have been easier to write this script in Python? I don't think it's worth rewriting it now though.

Reply:
I was thinking that as I finished writing it... but yea didn't seem worth rewriting.